### DISCUSSION TITLE
GPU-Z v2.58.0 now reports ReBar support correctly on Turing!

### DISCUSSION THREAD

**Discussion by TessellatedGuy** [terminatorul/NvStrapsReBar#78] [General]
The changelog contains this line:

<img width="512" alt="Changelog" src="https://github.com/terminatorul/NvStrapsReBar/assets/13260031/1d9babf1-f46e-40c3-b991-7d41d96e1a49">

The GPU-Z PCIe Resizable BAR page now shows everything as "Yes", on my RTX 2060:

<img width="291" alt="GPU-Z" src="https://github.com/terminatorul/NvStrapsReBar/assets/13260031/99e60340-c2d2-4edf-a0a7-c5d729d6eab4">

**Comment by terminatorul** [comment]
Thank you, this is always nice news to hear !
